CMakeLists.txt

```diff
@@ -86,25 +86,20 @@ else ()
         if (USE_OPENSSL AND NOT ANDROID)
             find_package(OpenSSL REQUIRED)
             find_package(Threads REQUIRED)
-            add_library(crypto UNKNOWN IMPORTED)
-            set_target_properties(crypto PROPERTIES
-                INTERFACE_INCLUDE_DIRECTORIES "${OPENSSL_INCLUDE_DIR}")
-            set_target_properties(crypto PROPERTIES
-                IMPORTED_LINK_INTERFACE_LANGUAGES "C"
-                IMPORTED_LOCATION "${OPENSSL_CRYPTO_LIBRARY}")
-            add_dependencies(crypto Threads::Threads)
+            set(PLATFORM_LIBS OpenSSL::Crypto Threads::Threads)
```

Comment:
why does this branch require "Threads" but the other branches don't?

Reply:
attempting to not change behavior, just fix the targets.